setup.py

```diff
@@ -16,6 +16,12 @@
       classifiers = [
                      'Development Status :: 4 - Beta',
                      'Programming Language :: Python',
+                     'Programming Language :: Python :: 2.7',
+                     'Programming Language :: Python :: 3',
+                     'Programming Language :: Python :: 3.4',
+                     'Programming Language :: Python :: 3.5',
+                     'Programming Language :: Python :: 3.6',
+                     'Programming Language :: Python :: 3.7',            
```

Comment:
Little nit, remove training space:
```suggestion
                     'Programming Language :: Python :: 3.7',
```